docs/setup_guide.md

```diff
@@ -52,17 +52,30 @@ For this guide we will assume that the returned template id is `0x00000000000000
 
 ### Deploying the module
 
-Now that we have a template, a hardhat task can be used to deploy a DAO module instance. This setup task requires the following parameters: `dao` (the address of the Safe), `oracle` (the address of the Realitio contract) and `template` (the template to be used with Realitio). There are also optional parameters, for more information run `yarn hardhat setup --help`.
+Hardhat tasks can be used to deploy a dao module instance. There are two different tasks, the first one is through a normal deployment and passing arguments to the constructor (with the task `setup`), or, deploy the Module through a [Minimal Proxy Factory](https://eips.ethereum.org/EIPS/eip-1167) and save on gas costs (with the task `factorySetup`) - In rinkeby the address of the Proxy Factory is: `0xd067410a85ffC8C55f7245DE4BfE16C95329D232` and the master copy of the DAO Module: `0x4D0D4Bd6eCA52f2F931c099B6a8a8B2ae85FFD4E`.
```

Comment:
Do we have a list of all addresses for different networks somewhere? Also do we have some information how the master copy was deployed?

Reply:
> Do we have a list of all addresses for different networks somewhere?

Yes! We do on the [zodiac repository](https://github.com/gnosis/zodiac/blob/master/src/factory/constants.ts), do you think it would make sense to add the link here?

> Also do we have some information how the master copy was deployed?

The information is on the bottom section of the doc - If you feel that we should mention it here we can do that

Reply:
Yes for now I would add a link here (additional to the examples) :)

> The information is on the bottom section of the doc
Might make sense to link to it, but not critical